superset-frontend/src/components/ListViewCard/index.tsx

```diff
@@ -0,0 +1,196 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import styled from '@superset-ui/style';
+import Icon from 'src/components/Icon';
+import { Card } from 'src/common/components';
+
+const MenuIcon = styled(Icon)`
+  width: 14px;
+  height: 14px;
+  position: relative;
+  top: 1px;
+`;
+
+const ActionsWrapper = styled.div`
+  width: 64px;
+  display: flex;
+  justify-content: space-between;
+`;
+
+const StyledCard = styled(Card)`
+  width: 459px;
```

Comment:
In general, it'd be good to use gridUnits, but this one jumped out at me... why 459 specifically? There's also a 264px height below on the Cover styles, which seems like it should be propped open by its contents.